dist/styles/main.css

```diff
@@ -4098,7 +4098,7 @@ to{background-color:transparent}
 }
 @keyframes fadeInProgress{to{background-color:#0088ce}
 }
-@keyframes fadeInAborted{to{background-color:#e8e8e8}
+@keyframes fadeInAborted{to{background-color:#d6d6d6}
```

Comment:
Should we use `d1d1d1` from the Patternfly palette?

https://www.patternfly.org/styles/color-palette/#_